### PR TITLE
Text and list components

### PR DESCRIPTION
**Pull request by samyukthasudhakar**
As part of this PR, have created the following components :

- text utilities
- lists

Live : https://text-and-lists.netlify.app/docs.html#text

Please let me know if the components look alright and needs any changes as well 😄

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
Looks good!

**Review by vrx29 — APPROVED** [review]
looks good to me

File: styles/components/list.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+.ul-square{
+    list-style-type: square;
+}
+
+.ul-no-style{
+    list-style-type: none;
+}
+
+.li-no-style{
+    margin: 1rem 0;
```
Comment (venkateshwarans) [inline styles/components/list.css]: is this intended?

**Review by venkateshwarans — APPROVED** [review]
Looks good.

File: styles/components/list.css (hunk 1 above)
Comment (samyukthasudhakar) [reply styles/components/list.css]: yes it is. i'll move this to utility. Thanks for pointing it out.

**Pull request merged**